src/lib/components/Notification.svelte

```diff
@@ -1,32 +1,44 @@
 <script lang="ts">
-  export let notificationTitle: string;
-  export let notificationText: string;
-  export let notificationLink: string;
+  import LiveTimeSince from "$lib/components/LiveTimeSince.svelte";
+  import type { NotificationSchema } from "$lib/zod/schemas";
+  import type { Notification } from "@prisma/client";
+  import type { SuperValidated } from "sveltekit-superforms";
+  import { superForm } from "sveltekit-superforms/client";
+  type NotificationItem = Pick<
+    Notification,
+    "link" | "title" | "message" | "createdAt" | "id"
+  >;
```

Comment:
Instead of manually declaring the `Notification` object, use the type export from `@prisma/client`. And even better, if you don't use all properties, use `Pick<...>` to select only the properties you actually use. This is nice in the future if we want to do optimization and at a glance see which properties we need to extract from the DB for each page.